wls-basisdaten-service/src/main/resources/db/migrations/h2/V6_0__createWahlTable.sql

```diff
@@ -1,15 +1,15 @@
-create table Wahl
+CREATE TABLE Wahl
 (
-    wahlid                      varchar(1024) not null,
-    name                        varchar(255) not null,
-    reihenfolge                 bigint not null,
-    waehlerverzeichnisnummer    bigint not null,
-    wahltag                     datetime not null,
-    wahlart                     varchar(255) not null,
-    r                           bigint,
-    g                           bigint,
-    b                           bigint,
-    nummer                      varchar(255) not null,
+    wahlID                      VARCHAR(1024) NOT NULL,
+    name                        VARCHAR(255) NOT NULL,
+    reihenfolge                 BIGINT NOT NULL,
+    waehlerverzeichnisnummer    BIGINT NOT NULL,
+    wahltag                     DATETIME NOT NULL,
```

Comment:
Warum ist zum Beispiel der `wahltag` manchmal `DATETIME` und manchmal `TIMESTAMP` ? Ich denke das sollte für alle Datums-/Zeitangaben einheitlich seineinheitlich sein. Ich weiß aber nicht, ob das in diesen PR gehört.

Reply:
gute Frage, würde ich aber nicht als Teils des PRs sehen

Reply:
> Warum ist zum Beispiel der `wahltag` manchmal `DATETIME` und manchmal `TIMESTAMP` ? Ich denke das sollte für alle Datums-/Zeitangaben einheitlich sein. Ich weiß aber nicht, ob das in diesen PR gehört.

Haben es so aus WLS2.0 übernommen. Ich weiss es aber nicht ob es notwendig ist.

Reply:
Wird in neuem Issue #493 behandelt